### ISSUE TITLE
need a PortalStandardPage to include search bar/help link and Portal footer linked pages 

### ISSUE THREAD

**Issue by MomoMoses** [johnjung/bmrcportal#109]
about and help currently do not display a searchbar at the top (with link to Research Help) nor the linked pages above the standard BMRC footer.
need to create a new standard page that includes these elements on every Portal page; otherwise it creates dead ends and user confusion (one expects).

**Comment by johnjung** [comment]
This is now implemented in https://github.com/uchicago-library/bmrc/commit/82aa13f57413356a4f8916f40f2c4efeb7e841be and live on bmrc-test.lib. Please check to be sure the site works the way you expect it to.

**Comment by MomoMoses** [comment]
NO - neither the "about" page nor the "help" page has the updated Standard Page that includes the search bar and the links above the footer. Please review.

**Comment by johnjung** [comment]
I reviewed this- this seems to be set up correctly.

Pages that already exist don't get converted to new content types in Wagtail- in situations like this, you would have to delete the existing "about" and "help" pages using the Wagtail admin and create new ones using the PortalStandardPage.

I made a sample page at https://bmrc-test.lib.uchicago.edu/portal/standard-page-test/ to confirm that the search bar and additional footer links are in place. Does the sample page work for you?
